tests/test_cli/test_interact.py

```diff
@@ -187,3 +188,63 @@ def test__process_envelopes_couldnt_recover(self, *mocks):
 
         with self.assertRaises(AssertionError):
             _process_envelopes(agent_name, identity_stub, inbox, outbox)
+
+
+class TestInteractEcho(AEATestCaseEmpty):
+    """Test 'aea interact' with the echo skill."""
+
+    @skip_test_windows
+    @pytest.mark.integration
+    def test_interact(self):
+        """Test the 'aea interact' command with the echo skill."""
+        self.add_item("skill", "fetchai/echo:0.4.0")
+        self.run_agent("--connections", "fetchai/stub:0.7.0")
```

Comment:
This agent only has stub connection, so I'd remove the specific flag for stub connection, means less string substitutions when we bump